File: skills/techs/aspnet-minimal-api.ts
export const aspnetMinimalApiTech = {
  id: 'aspnet-minimal-api',
  name: 'ASP.NET Minimal API',
  detect: {
    configFiles: ['appsettings.json'],
    configFileContent: { scanDotNetLayout: true, patterns: ['Microsoft.AspNetCore.OpenApi', 'Swashbuckle.AspNetCore'] },
  },
  skills: ['github/awesome-copilot/aspnet-minimal-api-openapi', 'dotnet/skills/minimal-api-file-upload'],
};

File: src/skills/techs/aspnet_minimal_api.rs
use crate::skills::types::{ConfigFileContentBlock, DetectConfig, Technology};

pub const ASPNET_MINIMAL_API_TECH: Technology = Technology {
    id: "aspnet-minimal-api",
    name: "ASP.NET Minimal API",
    detect: DetectConfig {
        packages: &[],
        package_patterns: &[],
        config_files: &["appsettings.json"],
        file_extensions: &[],
        gems: &[],
        config_file_content: &[ConfigFileContentBlock {
            files: &[],
            patterns: &["Microsoft.AspNetCore.OpenApi", "Swashbuckle.AspNetCore"],
            scan_gradle_layout: false,
            scan_dotnet_layout: true,
        }],
    },
    skills: &[
        "github/awesome-copilot/aspnet-minimal-api-openapi",
        "dotnet/skills/minimal-api-file-upload",
    ],
};
